assets/scss/_features.scss

```diff
@@ -0,0 +1,28 @@
+.features {
+    
+}
+
+.features__list {
+    background: $white;
+    padding: 20px;
+    box-shadow: 0px 2px 3px 0px $gray;
+    font-family: $montserrat;
+    font-size: $font-xs;
+    display: flex;
+}
+
+.features__item {
+    width: 33.3333%;
```

Comment:
you are using flex you can simply use `space-around` `space-between` or `space-evenly` in parent element and set width up there